#ifndef THOR_MANG_CONTROL_MODE_SWITCHER_H
#define THOR_MANG_CONTROL_MODE_SWITCHER_H

#include <ros/ros.h>


namespace control_mode_switcher{

    class ControlModeSwitcher {

    public:
      ControlModeSwitcher();
      virtual ~ControlModeSwitcher();
    protected:
     
    private:
     
    };
}


#endif // THOR_MANG_CONTROL_MODE_SWITCHER_H
